src/jtag/drivers/xds110.c

```diff
@@ -579,7 +579,7 @@ static bool usb_get_response(uint32_t *total_bytes_read, uint32_t timeout)
 
 static bool usb_send_command(uint16_t size)
 {
-	int written;
+	int written = 0;
```

Comment:
I assume this is to silence a compiler warning (or something similar), am I correct?

Reply:
Yes. I must be using a newer compiler or something, that now knows to look inside usb_write() and can tell that there is a return there before *written is ever assigned.